### DISCUSSION TITLE
cURL invoices not showing BTC amount

### DISCUSSION THREAD

**Discussion by LuckyKey613** [btcpayserver/btcpayserver#5880] [General]
Would anyone know a way I can retrieve settled invoices that show how much BTC was paid in the invoice? The amount was in USD but I need to know how much BTC was provided as payment. Thanks, in advance.

**Comment by dennisreimann** [comment]
You need to fetch the associated payment methods, that endpoint has the information you are looking for.

**Comment by LuckyKey613** [comment]
hmmm ?  Excuse the ignorance, but is that an easy do? Can you do that via cURL?

**Comment by LuckyKey613** [comment]
....looks like you can, just ran a successful payment-methods cURL request.

Thank you!
